### DISCUSSION TITLE
Bug with labels

### DISCUSSION THREAD

**Discussion by nt4f04uNd** [community/community#3899] [Mobile] [answered]
When activity is restored after being killed, labels disappear and issues list is displayed without being filtered

* In repo issue section that means that simply all issues/PRs will be displayed

* In personal - any persons issue/PR

Videos accordingly attached ("don't keep activities" enabled)

Ping @aconsuegra

https://user-images.githubusercontent.com/39104740/117319033-9b11e980-ae93-11eb-9e2e-18896f6b7b3c.mp4

https://user-images.githubusercontent.com/39104740/117319116-ae24b980-ae93-11eb-9276-4e1d6d95d32d.mp4

**Comment by aconsuegra** [comment, marked as answer]
Thanks for the report @nt4f04uNd! We didn't know about this bug, so we will look at it ASAP
